### ISSUE TITLE
Pretrained Weight

### ISSUE THREAD

**Issue by SunshineJunFu** [stevewongv/SPANet#14]
Hi, Could you release your pretrained weight on Rain1200 ? Look forward to your response.

**Comment by stevewongv** [comment]
We only provide the pre-trained model in our dataset. You can only compare our method in real cases like our 1000 real test set and the images from internet.

**Comment by SunshineJunFu** [comment]
Thanks!
